mne/io/tests/test_constants.py

```diff
@@ -0,0 +1,260 @@
+                        defines[t] = [s, d]
+                    else:
+                        assert not line.startswith('enum(')
+                else:  # in an enum
+                    if line == '{':
+                        continue
+                    elif line == '}':
+                        in_ = None
+                        continue
+                    t, s, d = re_enum_entry.match(line).groups()
+                    s = int(s)
+                    if t != 'ecg' and s != 3:  # ecg defined the same way
+                        assert s not in in_
+                    in_[s] = [t, d]
+
+    #
+    # Assertions
+    #
+
+    # Version
+    mne_version = '%d.%d' % (FIFF.FIFFC_MAJOR_VERSION,
+                             FIFF.FIFFC_MINOR_VERSION)
+    assert fiff_version == mne_version
+    unknowns = list()
+
+    # Assert that all our constants are in the dict
+    # (we are not necessarily complete the other way)
+    for name in sorted(dir(FIFF)):
+        if name.startswith('_') or name in _dir_ignore_names or \
+                name in _missing_names:
+            continue
+        val = getattr(FIFF, name)
+        if name in defines:
+            assert defines[name][0] == val
+        elif name.startswith('FIFFC_'):
+            # Checked above
+            assert name in ('FIFFC_MAJOR_VERSION', 'FIFFC_MINOR_VERSION',
+                            'FIFFC_VERSION')
+        elif name.startswith('FIFFB_'):
+            assert val in iod, (val, name)
+        elif name.startswith('FIFFT_'):
+            assert val in types, (val, name)
+        elif name.startswith('FIFFV_'):
+            if name.startswith('FIFFV_MNE_') and name.endswith('_ORI'):
+                this_enum = 'mne_ori'
+            elif name.startswith('FIFFV_MNE_') and name.endswith('_COV'):
+                this_enum = 'covariance_type'
+            elif name.startswith('FIFFV_MNE_COORD'):
+                this_enum = 'coord'  # weird wrapper
+            elif name.endswith('_CH') or '_QUAT_' in name or name in \
+                    ('FIFFV_DIPOLE_WAVE', 'FIFFV_GOODNESS_FIT',
+                     'FIFFV_HPI_ERR', 'FIFFV_HPI_G', 'FIFFV_HPI_MOV'):
+                this_enum = 'ch_type'
+            elif name.startswith('FIFFV_SUBJ_'):
+                this_enum = name.split('_')[2].lower()
+            else:
+                for this_enum in used_enums:
+                    if name.startswith('FIFFV_' + this_enum.upper()):
+                        break
+                else:
+                    raise RuntimeError('Could not find %s' % (name,))
+            assert this_enum in used_enums, name
+            assert val in enums[this_enum], (val, name)
+        elif name.startswith('FIFF_UNIT'):  # units and multipliers
+            this_enum = name.split('_')[1].lower()
+            assert val in enums[this_enum], (name, val)
+        elif name.startswith('FWD_'):
+            # XXX remove from FIFF to forward.py namespace at some point?
```

Comment:
@agramfort one more decision point: We currently pollute the `FIFF` namespace with `FIFF.FWD_` values that are not part of the FIFF spec and are not written to disk. Can we just make a new `FWD.` bunch in `mne/forward/forward.py` for these? This will break things if people use `FIFF.FWD_` in their own code somewhere but I doubt it's done.

Reply:
agreed.